core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -132,7 +132,105 @@ describe('Overview', () => {
             expect(jazzServices_po.getYearVerify().getText()).toEqual('YEAR');
             browser.sleep(5000);
             jazzServices_po.getServiceFromAsset().click();
-      });      
+      });
+
+      it('Verify METRICS Navigation for API' , () => {
+            // Navigation to services
+            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            //To Navigate to the particular service and verifying the Page
+            jazzServices_po.getAwsServiceName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            browser.sleep(5000);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+            jazzServices_po.getRefresh().click();
+            browser.sleep(2000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdName()), timeOutHigh);
+            jazzServices_po.getProdName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+            browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
+            browser.driver.switchTo().activeElement();
+            browser.sleep(5000);
+            jazzServices_po.getMetrices().click();
+            browser.driver.sleep(5000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getDeploymentStatus()), timeOutHigh);
+            jazzServices_po.getDeploymentStatus().click();
```

Comment:
line 157. what is the reason for the click? we are not checking anything in the following lines.

Reply:
metrics value takes some to reflect due this we have to refresh or click on different tab so that it will refresh the pages. this is the reason we are clicking on deployment tab.